fhir-ig-importer/importer/volume/ig-importer-app.json

```diff
@@ -3,7 +3,7 @@
   "description": "FHIR IG microfrontend app",
   "category": "HIE Configuration",
   "type": "esmodule",
-  "url": "http://localhost:3334/jembi-fhir-ig-importer.js",
+  "url": "https://openhimconsole.<domain>/fhir-ig-importer",
```

Comment:
can we still access this when running locally?

Reply:
Yes, but the domain should point to the console URL. Should I modify this to `<openhim-console-url>/fhir-ig-importer` @bradsawadye? Also, I have created a ticket that will solve the issue of hardcoded properties